src/chigame/games/views.py

```diff
@@ -196,6 +196,12 @@ class TournamentDeleteView(DeleteView):
     success_url = reverse_lazy("tournament-list")
 
 
+def TournamentChatDetailView(request, pk):
```

Comment:
This function doesn't feel like it's exclusive to tournaments, but could be generalized to matches. I think we should talk to the Matches team and see if individual matches should have a live chat.

Reply:
It seems per our chat model we are only assigning match as a foreign key to chat. We cannot really exclusively assign chat to each tournament but have to assign it to matches; Also for now, I think just per demonstration purpose it is fine and in later PRs in which we will "auto generate chat for each match (see issue #226 )", we will work on this if this is still a concern

Reply:
Sounds good to me! We can always revisit this if you want.